src/scripts/popup.js

```diff
@@ -295,6 +298,17 @@ function markAllAsRead() {
     markAsRead(feedIds);
 }
 
+function markAsReadEngagement() {
+    var feedIds = [];
+    $(".item:visible").each(function (key, value) {
+        var engagement = $(value).find("span.engagement").text() || 0; //default value if no engagement: 0
+        if(engagement <= popupGlobal.backgroundPage.appGlobal.options.engagementFilterLimit) {
```

Comment:
Probably we should change the condition to `less than`?
Because the option says: `Article to hide if engagement less than`.
In this case, we should change min value of the parameter to 1.